### PR TITLE
Priority deploy + Travis + JUnit

### PR DESCRIPTION
**Pull request by simon-andrews**
Boy oh boy!

Waiting on #14 to be approved so I can handle merging.

### CODE REVIEW

**Comment by Math-ias** [comment]
Merged #14 go ahead.

File: server/src/main/java/org/robockets/runqueue/server/Queue.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+		user.joinQueue(priority);
+	}
+	
+	/**
+	 * @return Next user in the queue.
+	 */
+	public static User next () {
+		boolean foundUser = false;
+		User u;
+		while(foundUser == false) {
```
Comment (jakekinsella) [inline server/src/main/java/org/robockets/runqueue/server/Queue.java:28]: Put a space between `while` and `(`. This goes for all `if`s and the like

File: server/src/main/java/org/robockets/runqueue/server/Queue.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+				return u;
+			}
+		}
+		return null;
+	}
+	
+	/**
+	 * @return {@link java.util.ArrayList} of every user that is in the queue
+	 */
+	public static ArrayList<User> getAllInQueue() {
```
Comment (jakekinsella) [inline server/src/main/java/org/robockets/runqueue/server/Queue.java:41]: Space between `getAllInQueue` and `(`. This goes for all functions like this

File: server/src/main/java/org/robockets/runqueue/server/UserPriorityComparator.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+	 * Comparator for checking if the priority of u1 is less than, greater than
+	 * or equal to the priority of user two.
+	 * @param u1 First user
+	 * @param u2 Second user
+	 */
+	@Override
+	public int compare (User u1, User u2) {
+		if (u1.getPriority().ordinal() < u2.getPriority().ordinal()) {
+			return -1;
+		}
```
Comment (jakekinsella) [inline server/src/main/java/org/robockets/runqueue/server/UserPriorityComparator.java:21]: Newline after the `}`

File: server/src/main/java/org/robockets/runqueue/server/UserPriorityComparator.java [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+		if (u1.getPriority().ordinal() > u2.getPriority().ordinal()) {
+			return 1;
+		}
```
Comment (jakekinsella) [inline server/src/main/java/org/robockets/runqueue/server/UserPriorityComparator.java:24]: Newline after the `}`

**Comment by jakekinsella** [comment]
Alright for now a load of styling changes. We will be implementing more Travis tests next Sprint

**Pull request merged**